package.json

```diff
@@ -7,6 +7,7 @@
     "scripts": {
         "build": "cross-env NODE_ENV=production lerna run build",
         "lint": "eslint --report-unused-disable-directives --max-warnings 0 --ext .js,.ts,.jsx,.tsx .",
+        "prettier": "prettier --write \"packages/**/*.{js,jsx,ts,tsx,css,scss,md}\"",
```

Comment:
Added a command for yarn prettier to .travis.yml

Reply:
A couple things,

- Having a script called "prettier" makes it so that if you call `yarn prettier`, it calls the script instead of calling the `node_modules/.bin/prettier`. Could we rename this script so it doesn't block they prettier CLI from being used by itself?

- Since eslint takes care of prettier for our js/ts files, maybe we should have prettier just check the css, scss, and md files. I also noticed that prettier is checking e.g. all the js files in our `build` folders.

- The travis command as written will end up with prettier being called with both `--write` and `--check`. In my testing, the `--write` behavior seems to override the `--check` behavior, and so even if there is an error, it ends up with a 0 exit code.

Reply:
was only able to do this by adding two subcommands for runprettier and checkprettier. I agree on the ts/js/tsx/jsx removal

Reply:
One more thing, if we only use it on css, scss, and md files, then we can probably safely run it on all files and not just those in `packages/`, so the files like those in the root or `docs/` don't get skipped.